### PR TITLE
Re-generate Kaptain docs

### PR DESCRIPTION
**Pull request by alembiewski**
Signed-off-by: Alex Lembiyeuski <alembiyeuski@d2iq.com>

## Description of changes being made
Update Kaptain generated documentation

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4565.s3-website-us-west-2.amazonaws.com/

## Checklist

- [x] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by gvasquezvargas — APPROVED** [review]

File: pages/dkp/kaptain/2.1.0/configuration/index.md [hunk 1]
```diff
@@ -31,20 +32,21 @@ This page lists Kaptain configuration properties that can be adjusted during the
...
+| core.notebook.images[0] | string | `"mesosphere/kubeflow-dev:9715a1de-jupyter-spark-3.3.0-horovod-0.24.2-tensorflow-2.8.0"` | JupyterLab with Tensorflow, Spark and Horovod pre-installed |
+| core.notebook.images[1] | string | `"mesosphere/kubeflow-dev:14894f36-jupyter-spark-3.3.0-horovod-0.24.2-tensorflow-2.8.0-gpu"` | JupyterLab with Tensorflow, CUDA, Spark and Horovod pre-installed with GPU support |
+| core.notebook.images[2] | string | `"mesosphere/kubeflow-dev:58054fea-jupyter-spark-3.3.0-horovod-0.24.2-pytorch-1.11.0"` | JupyterLab with Pytorch, Spark and Horovod pre-installed |
+| core.notebook.images[3] | string | `"mesosphere/kubeflow-dev:ecfaaddd-jupyter-spark-3.3.0-horovod-0.24.2-pytorch-1.11.0-gpu"` | JupyterLab with Pytorch, CUDA, Spark and Horovod pre-installed with GPU support |
+| core.notebook.images[4] | string | `"mesosphere/kubeflow-dev:eed5f454-jupyter-spark-3.3.0-horovod-0.24.2-mxnet-1.9.0"` | JupyterLab with MXNet, Spark and Horovod pre-installed |
+| core.notebook.images[5] | string | `"mesosphere/kubeflow-dev:75dd8b53-jupyter-spark-3.3.0-horovod-0.24.2-mxnet-1.9.0-gpu"` | JupyterLab with MXNet, CUDA, Spark and Horovod pre-installed with GPU support |
```
Comment (MessKon) [inline pages/dkp/kaptain/2.1.0/configuration/index.md:41]: are you using a script to come up with these image version values?

File: pages/dkp/kaptain/2.1.0/configuration/index.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ This page lists Kaptain configuration properties that can be adjusted during the
 | ingress.enabled | bool | `true` | Enable ingress |
 | ingress.externalDexClientId | string | `""` | Dex client ID to use when authenticating with external cluster |
 | ingress.externalDexClientSecret | string | `""` | Dex client secret to use when authenticating with external cluster |
+| ingress.oidcGroupsAllowList | string | `"*"` | List of groups that are allowed to pass authorization |
```
Comment (MessKon) [inline pages/dkp/kaptain/2.1.0/configuration/index.md:25]: ✅

File: pages/dkp/kaptain/2.1.0/configuration/index.md (hunk 1 above)
Comment (alembiewski) [reply pages/dkp/kaptain/2.1.0/configuration/index.md:41]: yes, `make generate-docs` takes care of it

File: pages/dkp/kaptain/2.1.0/tutorials/training/mxnet/index.md [hunk 3]
```diff
@@ -154,9 +155,9 @@ mnist["train_label"][42]
 
 
 
-```sh
+
```
[the remark this answers is not in the document]
Comment (alembiewski) [reply pages/dkp/kaptain/2.1.0/tutorials/training/mxnet/index.md:158]: I don't know, maybe they were updated manually previously? Anyway, I checked the [preview](http://docs-d2iq-com-pr-4565.s3-website-us-west-2.amazonaws.com/dkp/kaptain/2.1.0/tutorials/training/mxnet/) and it looks good to me, so it seems like it doesn't impact the docs rendering in any way

**Pull request merged**